src/components/MagPage/MagPage.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'; // gimme React
+import classNames from 'classnames/bind'; // gotta name more than one class youngblood
+import PreviewRow from './PreviewRow'; // Imports the Mag Previews component
+import styles from './styles/magazinePage.css'; // show me my stylesheets
+
+let cx = classNames.bind(styles);
+
+class MagPage extends React.Component {
+    render() {
+        return (
+
+        	<div>
+            	<img className={cx("bkgImage")} src="/img/misc/bedroom.png" />
+            	<div className = {cx("headerText")}>
+	            	<h1> MAGAZINES </h1>
+	            	<h4> READ THE NEWEST SUMMER 16 MAG! <br/><br/><br/></h4>
```

Comment:
I believe that a switch to `margin-bottom` or `padding-bottom` for vertical spacing would help here